ui/components/multichain/pages/review-permissions-page/__snapshots__/review-permissions-page.test.tsx.snap

```diff
@@ -44,7 +44,7 @@ exports[`ReviewPermissions should render correctly 1`] = `
                 class="mm-box mm-text mm-text--heading-md mm-text--ellipsis mm-text--text-align-center mm-box--color-text-default"
               />
             </div>
-          </p>
+          </div>
```

Comment:
There’s no Storybook story for this page, and I’m not sure where it is in the app off the top of my head, but based on the other screenshots, I’m confident there are no visual regressions.